### PR TITLE
feat: Scripture node connected

### PR DESCRIPTION
**Pull request by vinnyjth**
## DESCRIPTION

Simulator Screen Shot - iPhone 11 - 2020-12-02 at 15 09 07

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      fetchPolicy={'cache-and-network'}
+    >
+      {({
+        data: { node: { scriptures = [] } = { scriptures: [] } } = {},
+        loading,
+        error,
+      }) => {
+        // We don't want to show a loading state since this is an optional component.
+        // However, if we know there are scriptures (from cached data) we should go ahead and show scripture
+        console.log(scriptures, loading, error);
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js:23]:
```suggestion
```

**Comment by redreceipt** [comment]
this isn't wired to anything yet correct? just the story?

**Comment by vinnyjth** [comment]
Correct. There's another blocked PR that uses this code however

**Comment by vinnyjth** [comment]
@redreceipt you can see the comment out lines here where it will be used: https://github.com/ApollosProject/apollos-apps/pull/1660

File: packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Query } from '@apollo/client/react/components';
+import ScriptureItem from '@apollosproject/ui-scripture';
+import { PaddedView } from '@apollosproject/ui-kit';
+import GET_SCRIPTURE_NODE from './getScriptureNode';
+
+const ContentNodeConnected = ({ nodeId }) => {
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js:8]:
```suggestion
const ScriptureNodeConnected = ({ nodeId }) => {
```

File: packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  if (!nodeId) return <ScriptureItem isLoading />;
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js:9]: shouldn't we show an error here if a `nodeId` isn't passed?

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js:9]: `nodeId` might not be passed (or might be null) if the parent component is loading, and doesn't have one yet.

File: packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+              />
+            ))}
+          </PaddedView>
+        );
+      }}
+    </Query>
+  );
+};
+
+ContentNodeConnected.propTypes = {
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js:47]:
```suggestion
ScriptureNodeConnected.propTypes = {
```

File: packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  nodeId: PropTypes.string.isRequired,
+};
+
+ContentNodeConnected.defaultProps = {};
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js:51]:
```suggestion
ScriptureNodeConnected.defaultProps = {};
```

File: packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+export default ContentNodeConnected;
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ScriptureNodeConnected/ScriptureNodeConnected.js:53]:
```suggestion
export default ScriptureNodeConnected;
```

**Review by redreceipt — APPROVED** [review]
looks great

**Pull request merged**